docs/snippets/ov_preprocessing.py

```diff
@@ -169,3 +169,56 @@ def custom_abs(output: Output):
 ppp.output("result_image").postprocess()\
     .custom(custom_abs)
 # ! [ov:preprocess:postprocess]
+
+# ! [ov:preprocess:save_headers]
+from openvino.preprocess import PrePostProcessor, ColorFormat, ResizeAlgorithm
+from openvino.runtime import Core, Layout, Type, set_batch
+from openvino.offline_transformations import serialize
```

Comment:
`offline_transformations` is not a official public API, do we have an ability to serialize via public API?

Reply:
Yes, we can serialize using pass_manager:
```python    
pass_manager = Manager()\n
pass_manager.register_pass(pass_name="Serialize", xml_path=xml_path, bin_path=bin_path)
```

Reply:
Fixed, thanks